test/jdk/java/awt/Focus/ComponentLostFocusTest.java

```diff
@@ -100,25 +95,16 @@ public void run() {
         // We want to track the GAIN_FOCUS from this time
         tf.addFocusListener(new FocusAdapter() {
             public void focusGained(FocusEvent e) {
-                System.out.println("TextField gained focus: "+e);
+                System.out.println("TextField gained focus: " + e);
                 passed = true;
             }
         });
 
-        doRequestFocusToTextField();
-
-        System.out.println("Focused window: " + KeyboardFocusManager.getCurrentKeyboardFocusManager().getFocusedWindow());
-        System.out.println("Focus owner: " + KeyboardFocusManager.getCurrentKeyboardFocusManager().getFocusOwner());
-
-        if (!passed) {
-            throw new RuntimeException("TextField got no focus! Test failed.");
-        }
     }
 
-    private void doRequestFocusToTextField(){
+    private static void doRequestFocusToTextField() {
         // do activation using press title
-        Point loc = frame.getLocationOnScreen();
-        r.mouseMove(loc.x + frame.getWidth()/2, loc.y + frame.getInsets().top/2);
+        r.mouseMove(loc.x + frame.getWidth() / 2, loc.y + frame.getInsets().top / 2);
```

Comment:
`frame.getWidth` also should be on EDT ?

Reply:
ok